modules/src/news.py

```diff
@@ -0,0 +1,22 @@
+import re
+import requests
+
+# Also not sure exactly what entities is?
```

Comment:
Entities will return relevant text from the query, try `define a superhero` in try out a sentence at the top [here](https://wit.ai/swapagarwal/JARVIS-on-Messenger).